### PR TITLE
bpo-23702: Update Descriptor-HOWTO to reflect the removal of unbound …

### PR DESCRIPTION
**Pull request by rhettinger**
https://bugs.python.org/issue23702

### CODE REVIEW

**Comment by nedbat** [comment]
Maybe now would be a good time to take a look at https://bugs.python.org/issue17894 ?

File: Doc/howto/descriptor.rst [hunk 1]
```diff
@@ -266,22 +266,23 @@ Python's object oriented features are built upon a function based environment.
 Using non-data descriptors, the two are merged seamlessly.
 
 Class dictionaries store methods as functions.  In a class definition, methods
-are written using :keyword:`def` and :keyword:`lambda`, the usual tools for
-creating functions.  The only difference from regular functions is that the
+are written using :keyword:`def` or :keyword:`lambda`, the usual tools for
+creating functions.  Methods only differ from regular functions in that the
```
Comment (serhiy-storchaka) [inline Doc/howto/descriptor.rst:270]: I'm not sure about "in that".

File: Doc/howto/descriptor.rst [hunk 2]
```diff
@@ -291,25 +292,34 @@ Running the interpreter shows how the function descriptor works in practice::
     ...         return x
```
Comment (serhiy-storchaka) [inline Doc/howto/descriptor.rst:292]: `(object)` above can be removed.

**Review by serhiy-storchaka — APPROVED** [review]
I'm not competent in reviewing English, but overall look good to me. Except one suspicious place.

**Comment by serhiy-storchaka** [comment]
> Maybe now would be a good time to take a look at https://bugs.python.org/issue17894 ?

Sure. Look also at https://bugs.python.org/issue26103.

**Pull request merged**